lib/Makefile.inc

```diff
@@ -205,6 +205,7 @@ LIB_CFILES =         \
   telnet.c           \
   tftp.c             \
   timeval.c          \
+  timeutil.C         \
```

Comment:
s/C/c/

Reply:
Thanks, also noticed this due to the many CI failures.